content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -539,6 +539,12 @@ The `olm.bundle.object` property [cue][cuelang-spec] schema is:
 [semver]: https://semver.org/spec/v2.0.0.html
 [semver-range]: https://github.com/blang/semver/blob/master/README.md#ranges
 
+#### `olm.deprecated`
+
+The `olm.deprecated` property is handled on the `olm.bundle` object such that if this property exists on the bundle object, the bundle is to be ignored and not installed. So, bundle deprecation was typically handled based on this property being present on the `olm.bundle` object. However, this was revelant for SQLite based catalogs which also required valid upgrade edges to be present to prevent foreign key violations.
+
+Similarly, the same goal is achieved for file-Based Catalogs simply by not including the bundle in the catalog; which effectively means that the `opm render` command will now exclude bundles that contain this property on the `olm.bundle` object. Although, there will be valid upgrades edges present even though the deprecated bundle and its metadata are excluded from the rendered file-Based Catalog.
```

Comment:
nit: Capitalize File in "File-Based Catalogs" to be consistent with other docs

Reply:
also, small typo: upgrades -> upgrade

Reply:
Thanks for the review @oceanc80, addressed your review feedback.